x-pack/plugins/observability_solution/logs_explorer/common/constants.ts

```diff
@@ -42,14 +44,24 @@ export const RESOURCE_FIELD = 'resource';
 export const DATA_GRID_COLUMN_WIDTH_SMALL = 240;
 export const DATA_GRID_COLUMN_WIDTH_MEDIUM = 320;
 
+export const RESOURCE_FIELD_CONFIGURATION: SmartFieldGridColumnOptions = {
+  type: 'smart-field',
+  name: RESOURCE_FIELD,
+  fallbackFields: ['host.name', 'service.name'],
```

Comment:
**nit**: we could use the constants already defined above in this file for the fallback fields.